build/linux/channels.gni

```diff
@@ -0,0 +1,23 @@
+assert(is_linux)
+
+brave_linux_channel_stable = ""
+brave_linux_channel_dev = "dev"
+linux_channel_stable = "stable"
+linux_channel_beta = "beta"
+linux_channel_dev = "unstable"
+linux_channel_nightly = "nightly"
```

Comment:
can we change this to `brave_linux_channel_nightly` which is not in upstream?

Reply:
done.